sda/cmd/api/swagger_v1.yml

```
@@ -0,0 +1,410 @@
+openapi: 3.0.3
+info:
+  title: SDA admin API
+  version: "1.0"
+  description: This is the admin API for the sensitive data archive.
+paths:
+  /c4gh-keys/add:
+    post:
+      description: Registeres an crypt4gh public key hash in the database
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/C4ghKeyAdd"
+      responses:
+        "200":
+          description: Successful operation.
```

```suggestion
          description: Successful operation
```
For consistency with section below. The same goes for line 21-23